service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.events.CaseManagementOrderIssuedEvent;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class CaseManagementOrderIssuedEventHandler {
+    private final CaseManagementOrderCaseLinkNotificationHandler caseManagementOrderCaseLinkNotificationHandler;
```

Comment:
again with what I was saying about context, for this class, they could just be `caseLinkNotificationHandler` and `documentLinkNotificationHandler`